tests/test_init.py

```diff
@@ -53,3 +57,60 @@ def test_parse_bad_patchwork_url(self):
             "Can't parse patchwork url: '{}'".format(patchwork_url)
             in context.exception
         )
+
+    def test_ktree_init(self):
+        """Test ktree.__init__ without specifying a work directory"""
+        result = skt.ktree(GIT_URI)
+        self.assertIsInstance(result, skt.ktree)
+        self.assertEqual(result.uri, GIT_URI)
+
+    def test_ktree_init_workdir(self):
+        """Test ktree.__init__ with specifying a work directory"""
+        workdir = tempfile.mkdtemp()
+        result = skt.ktree(GIT_URI, wdir=workdir)
+        self.assertIsInstance(result, skt.ktree)
+        self.assertEqual(result.uri, GIT_URI)
+        self.assertEqual(result.wdir, workdir)
+
+    def test_ktree_create_workdir(self):
+        """Ensure create_workdir() creates a work dir if it does not exist"""
+        workdir = tempfile.mkdtemp()
+        os.rmdir(workdir)
+        result = skt.ktree(GIT_URI, wdir=workdir)
+        self.assertIsInstance(result, skt.ktree)
+        self.assertEqual(result.uri, GIT_URI)
+        self.assertEqual(result.wdir, workdir)
+        self.assertTrue(os.path.isdir(workdir))
+
+    def test_ktree_remove_mergelog(self):
+        """Ensure remove_mergelog really removes the merge.log"""
+        workdir = tempfile.mkdtemp()
+        merge_log = "{}/merge.log".format(workdir)
+        with open(merge_log, 'w') as fileh:
+            fileh.write("merge log sample content")
+
+        result = skt.ktree(GIT_URI, wdir=workdir)
+        self.assertIsInstance(result, skt.ktree)
+        self.assertEqual(result.uri, GIT_URI)
+        self.assertEqual(result.wdir, workdir)
+        self.assertFalse(os.path.isfile(merge_log))
```

Comment:
I think, first of all, these two functions shouldn't be a part of the public interface (or exist at all). Then they're so trivial in purpose, they're not likely to fail and so should have tests for them written last, if at all.